addons/crm/tests/test_crm_lead_convert_mass.py

```diff
@@ -174,7 +174,7 @@ def test_mass_convert_performances(self):
         user_ids = self.assign_users.ids
 
         # randomness: at least 1 query
-        with self.assertQueryCount(user_sales_manager=1442):  # crm ??
+        with self.assertQueryCount(user_sales_manager=1600):  # crm ??
```

Comment:
This one is a major pain point. Probabilities are recomputed multiple times once partner is set, due to the inverse added in this commit, for some reason and I couldn't wrestle the ORM out of it.

Worst case we can move the "compute" logic to create/write but it's unfortunate :/

Reply:
The actual flow in the ORM, as far as I could tell, is:

- set partner
-> contact_partner is set by inverse
-> partner is computed based on contact_partner (doesn't change, obviously)
-> because partner "changed", phone and email changed
-> trigger the compute once for partner_id, once for phone and once for email

So in a way it's expected but it doesn't really make much sense in practice